paddle/framework/channel_test.cc

```diff
@@ -60,13 +60,52 @@ TEST(Channel, SufficientBufferSizeDoesntBlock) {
   delete ch;
 }
 
-TEST(Channel, SendOnClosedChannelPanics) {
-  const size_t buffer_size = 10;
-  auto ch = MakeChannel<size_t>(buffer_size);
-  size_t i = 5;
-  EXPECT_EQ(ch->Send(&i), true);  // should not block or panic
+// This tests that a  channel must return false
+// on send and receive performed after closing the channel.
+// Receive will only return false after close when queue is empty
+void ClosedChannelSendReceivePanics(Channel<size_t> *ch) {
```

Comment:
Is `SendReceiveWithACloseChannelShouldPanic` a more accurate name for this test?

Reply:
Fixed. Thank you.